evalml/data_checks/default_data_checks.py

```diff
@@ -25,12 +30,15 @@ class DefaultDataChecks(DataChecks):
         - `ClassImbalanceDataCheck` (for classification problem types)
         - `DateTimeNaNDataCheck`
         - `NaturalLanguageNaNDataCheck`
-        - `TargetDistributionDataCheck`
+        - `TargetDistributionDataCheck` (for regression problem types)
+        - `EqualIntervalDataCheck` (for time series problem types)
```

Comment:
The class is called `DateTimeFormatDataCheck` no?

Reply:
Ah thanks for catching that